rotkehlchen/exchanges/manager.py

```diff
@@ -283,6 +283,9 @@ def initialize_exchanges(
         self.database = database
         # initialize exchanges for which we have keys and are not already initialized
         for location, credentials_list in exchange_credentials.items():
+            if location not in SUPPORTED_EXCHANGES:
```

Comment:
add a comment here explaining what this does
```suggestion
            if location not in SUPPORTED_EXCHANGES:  # in case a no longer supported exchange key is in the DB
```